frontend/app/controllers/Event.scala

```diff
@@ -111,74 +110,72 @@ trait Event extends Controller with ActivityTracking {
       event.socialImgUrl,
       Some(event.schema)
     )
-    Ok(views.html.event.page(event, pageInfo))
-  }
-
-  def masterclasses = CachedAction { implicit request =>
-    val pageInfo = PageInfo(
-      CopyConfig.copyTitleMasterclasses,
-      request.path,
-      Some(CopyConfig.copyDescriptionMasterclasses)
-    )
-    Ok(views.html.event.masterclass(pageInfo, masterclassEvents.events))
-  }
-
-  def masterclassesByTag(rawTag: String, rawSubTag: String = "") = CachedAction { implicit request =>
-    val tag = MasterclassEvent.decodeTag( if(rawSubTag.nonEmpty) rawSubTag else rawTag )
-    val pageInfo = PageInfo(
-      CopyConfig.copyTitleMasterclasses,
-      request.path,
-      Some(CopyConfig.copyDescriptionMasterclasses)
-    )
-    Ok(views.html.event.masterclass(
-      pageInfo,
-      masterclassEvents.getTaggedEvents(tag),
-      MasterclassEvent.decodeTag(rawTag),
-      MasterclassEvent.decodeTag(rawSubTag)
-    ))
+    Ok(views.html.event.eventDetail(pageInfo, event))
   }
 
   private def chronologicalSort(events: Seq[model.RichEvent.RichEvent]) = {
     events.sortWith(_.event.start < _.event.start)
   }
-
-  def list = CachedAction { implicit request =>
+  
+  def eventsList = CachedAction { implicit request =>
```

Comment:
Yeah agreed, maybe I only need to rename the template not the controller method.